src/adapter/src/catalog/storage.rs

```diff
@@ -1226,24 +1426,8 @@ impl<'a> Transaction<'a> {
     }
 
     pub fn insert_user_role(&mut self, role: SerializedRole) -> Result<RoleId, Error> {
-        self.insert_role(role, USER_ROLE_ID_ALLOC_KEY, RoleId::User)
-    }
-
-    fn insert_system_role(&mut self, role: SerializedRole) -> Result<RoleId, Error> {
-        self.insert_role(role, SYSTEM_ROLE_ID_ALLOC_KEY, RoleId::System)
```

Comment:
Should `SYSTEM_ROLE_ID_ALLOC_KEY` be deleted (and maybe removed from the stash) if it's no longer being used?

Reply:
Good point, I've removed this from the stash and code.